environment.yml

```diff
@@ -15,7 +15,7 @@ dependencies:
   - numpy
   - pip
   - pip:
-      - git+https://github.com/modflowpy/flopy.git
+      - https://github.com/emorway-usgs/flopy/archive/update_model_splitter.zip
```

Comment:
Was going to merge this PR, but I think this needs to be fixed first.

Reply:
@langevin-usgs, yes, `environment.yml` needs to go back to the original, eventually.  It was a temporary insertion to determine whether the fix that was installed on that flopy branch would get one of the parallel autotest on this PR to pass, and as 2f83e5c show, it did.  The autotest that failed uses a hard-coded reference to `iuzno`, a reference that this PR is trying to get away from.  

After opening a PR on FloPy that would make the change to model_splitter.py permanent (and that was originally implement on a branch of my fork: https://github.com/emorway-usgs/flopy/archive/update_model_splitter), something is failing over on FloPy.  I need to get that PR ([#1936](https://github.com/modflowpy/flopy/pull/1963) sorted out, and once that is brought in to FloPy, I can undo the change in question (2f83e5c) and then this PR will be ready to go.

Reply:
After [#1963](https://github.com/modflowpy/flopy/pull/1963) was officially adopted on the FloPy repo, I was able to go back in this PR's history to the commit before the change shown in this conversation's history (see screen grab below), such that a1bd15b no longer shows up. In other words, since a1bd15b revealed that the changes proposed in what became #1963 were successful, the need for a1bd15b was effectively undone by #1963's official adoption.
![history](https://github.com/MODFLOW-USGS/modflow6/assets/3236576/cd35b2e2-0fdd-486a-a0b3-5ce457dd7960)